sass/layout/_headers.scss

```diff
@@ -9,6 +9,7 @@
     font-size: 14px;
     position: relative;
     width: 100%;
+    fill: rgba(var(--center-channel-color-rgb), 0.6);
```

Comment:
What purpose does this serve? The original `changeCss` doesn't include this one.